joplin/base/wagtail_hooks.py

```diff
@@ -15,7 +15,7 @@
 from html.parser import HTMLParser
 
 import wagtail.admin.rich_text.editors.draftail.features as draftail_features
-from wagtail.admin.rich_text.converters.html_to_contentstate import BlockElementHandler
+from wagtail.admin.rich_text.converters.html_to_contentstate import BlockElementHandler, InlineStyleElementHandler
```

Comment:
Where are we using `InlineStyleElementHandler`?

Reply:
ahh we aren't atm, i'll remove it (i was playing around with something else)